src/templates/haml/home/index.html.haml

```diff
@@ -38,34 +44,34 @@
       %p.center= l("Trade in the world's financial markets in a simple, user-friendly platform.")
       .grd-grid-7.grd-grid-mobile-12.grd-parent
-        %img.responsive.ja-hide{:alt => "", :src => url_for("images/pages/tour/trade-tutorial_1.svg")}
-        %img.responsive.hidden.ja-show{:alt => "", :src => url_for("images/pages/tour/tour-jp.svg")}
+        %img.responsive{:alt => "", :src => url_for("images/pages/tour/trade-tutorial_1.svg")}
       .grd-grid-5.grd-grid-mobile-12.grd-parent
         %ul
           %li!= l('<span>All markets:</span> currencies, stock indices, and commodities.')
           %li!= l('<span>All market conditions:</span> up/down, touch/no-touch, stays in/goes out.')
           %li!= l('<span>All durations:</span> from 10 seconds to 365 days.')
           %li!= l('<span>All payouts:</span> from $1 to $100,000.')
-#xreason
-  .grd-container
-    .grd-row-padding
-      %h1.center-aligned.xhead= l('Why choose us')
-      .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-12
-        %ul
-          %li.xlist1
-            %h3= l('Sharp prices')
-            %p= l("Binary.com's patented pricing technology allows you to benefit from the same rates of return as traders in the interbank market.")
-          %li.xlist2
-            %h3= l('Your funds are safe')
-            %p= l('With Binary.com, your money is secure. We segregate and hold client funds in secure AAA-rated banks.')
-      .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-12
-        %ul
-          %li.xlist3
-            %h3= l('Trade your way')
-            %p= l("Don't limit yourself! Whatever your market view, Binary.com allows you to put your strategy to work and profit from your predictions.")
-          %li.xlist4
-            %h3= l('Go with the leader')
-            %p= l('Binary.com is the award-winning industry pioneer in online options trading.')
+#xcta
+  #xreason
+    .grd-container
+      .grd-row-padding
+        %h1.center-aligned.xhead= l('Why choose us')
+        .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-12
+          %ul
+            %li.xlist1
+              %h3= l('Sharp prices')
+              %p= l("Binary.com's patented pricing technology allows you to benefit from the same rates of return as traders in the interbank market.")
+            %li.xlist2
+              %h3= l('Your funds are safe')
+              %p= l('With Binary.com, your money is secure. We segregate and hold client funds in secure AAA-rated banks.')
+        .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-12
+          %ul
+            %li.xlist3
+              %h3= l('Trade your way')
+              %p= l("Don't limit yourself! Whatever your market view, Binary.com allows you to put your strategy to work and profit from your predictions.")
+            %li.xlist4
+              %h3= l('Go with the leader')
+              %p= l('Binary.com is the award-winning industry pioneer in online options trading.')
```

Comment:
just adding a note for future that instead of Binary.com it should be stash->{website}